libindy/src/utils/logger.rs

```diff
@@ -5,7 +5,7 @@ extern crate log;
 extern crate android_logger;
 
 use self::env_logger::Builder;
-use self::log::LevelFilter;
+use self::log::{Record, LevelFilter, Level};
```

Comment:
Should we switch default log level for android from `Trace` to `Info` or `Off` below?
https://github.com/hyperledger/indy-sdk/pull/959/files#diff-5d11eb6aa67bd3369d836a1a7f0ca492R23